Get client.Client in webhook admission

Now i create webhook admission using `kubebuilder alpha webhook ...`, and then find the webhook 
handler generated like this:
```
type MyResourceCreateUpdateHandler struct {
	// Client  client.Client

	// Decoder decodes objects
	Decoder types.Decoder
}
```

The `Client` which i need in webhook validating or mutating has been commented.
How could i get client or informer in webhook admission?

When I run the following command. 
```kubebuilder alpha webhook --group core --version v1 --kind Pod --type=mutating --operations=create,update```

I get the following error : 
```pkg/webhook/add_default_server.go:24:2: undefined: AddToManagerFuncs
make: *** [vet] Error 2```

--- a/pkg/scaffold/webhook/admissionhandler.go
+++ b/pkg/scaffold/webhook/admissionhandler.go
@@ -99,6 +99,10 @@ func init() {
 
 // {{ .Resource.Kind }}{{ .OperationsString }}Handler handles {{ .Resource.Kind }}
 type {{ .Resource.Kind }}{{ .OperationsString }}Handler struct {
+	// To use the client, you need to do the following:
+	// - uncomment it
+	// - import sigs.k8s.io/controller-runtime/pkg/client
+	// - uncomment the InjectClient method at the bottom of this file.
 	// Client  client.Client
 
 	// Decoder decodes objects

--- a/test/project/pkg/webhook/default_server/firstmate/mutating/firstmate_create_update_handler.go
+++ b/test/project/pkg/webhook/default_server/firstmate/mutating/firstmate_create_update_handler.go
@@ -36,6 +36,10 @@ func init() {
 
 // FirstMateCreateUpdateHandler handles FirstMate
 type FirstMateCreateUpdateHandler struct {
+	// To use the client, you need to do the following:
+	// - uncomment it
+	// - import sigs.k8s.io/controller-runtime/pkg/client
+	// - uncomment the InjectClient method at the bottom of this file.
 	// Client  client.Client
 
 	// Decoder decodes objects

--- a/test/project/pkg/webhook/default_server/firstmate/mutating/firstmate_delete_handler.go
+++ b/test/project/pkg/webhook/default_server/firstmate/mutating/firstmate_delete_handler.go
@@ -36,6 +36,10 @@ func init() {
 
 // FirstMateDeleteHandler handles FirstMate
 type FirstMateDeleteHandler struct {
+	// To use the client, you need to do the following:
+	// - uncomment it
+	// - import sigs.k8s.io/controller-runtime/pkg/client
+	// - uncomment the InjectClient method at the bottom of this file.
 	// Client  client.Client
 
 	// Decoder decodes objects

--- a/test/project/pkg/webhook/default_server/frigate/validating/frigate_update_handler.go
+++ b/test/project/pkg/webhook/default_server/frigate/validating/frigate_update_handler.go
@@ -36,6 +36,10 @@ func init() {
 
 // FrigateUpdateHandler handles Frigate
 type FrigateUpdateHandler struct {
+	// To use the client, you need to do the following:
+	// - uncomment it
+	// - import sigs.k8s.io/controller-runtime/pkg/client
+	// - uncomment the InjectClient method at the bottom of this file.
 	// Client  client.Client
 
 	// Decoder decodes objects

--- a/test/project/pkg/webhook/default_server/kraken/validating/kraken_create_handler.go
+++ b/test/project/pkg/webhook/default_server/kraken/validating/kraken_create_handler.go
@@ -36,6 +36,10 @@ func init() {
 
 // KrakenCreateHandler handles Kraken
 type KrakenCreateHandler struct {
+	// To use the client, you need to do the following:
+	// - uncomment it
+	// - import sigs.k8s.io/controller-runtime/pkg/client
+	// - uncomment the InjectClient method at the bottom of this file.
 	// Client  client.Client
 
 	// Decoder decodes objects

--- a/test/project/pkg/webhook/default_server/namespace/mutating/namespace_update_handler.go
+++ b/test/project/pkg/webhook/default_server/namespace/mutating/namespace_update_handler.go
@@ -36,6 +36,10 @@ func init() {
 
 // NamespaceUpdateHandler handles Namespace
 type NamespaceUpdateHandler struct {
+	// To use the client, you need to do the following:
+	// - uncomment it
+	// - import sigs.k8s.io/controller-runtime/pkg/client
+	// - uncomment the InjectClient method at the bottom of this file.
 	// Client  client.Client
 
 	// Decoder decodes objects

